x-pack/plugins/security_solution/public/management/cypress/tasks/login.ts

```diff
@@ -99,6 +101,76 @@ export const constructUrlWithUser = (user: User, route: string) => {
   return builtUrl.href;
 };
 
+export const getCurlScriptEnvVars = () => ({
+  ELASTICSEARCH_URL: Cypress.env('ELASTICSEARCH_URL'),
+  ELASTICSEARCH_USERNAME: Cypress.env('ELASTICSEARCH_USERNAME'),
+  ELASTICSEARCH_PASSWORD: Cypress.env('ELASTICSEARCH_PASSWORD'),
+  KIBANA_URL: Cypress.config().baseUrl,
+});
+
+export const postRoleAndUser = (role: ROLE) => {
+  const env = getCurlScriptEnvVars();
+
+  // post the role
+  cy.request({
+    method: 'PUT',
+    url: `${env.KIBANA_URL}/api/security/role/${role}`,
+    body: rolesMapping[role],
+    headers: {
+      'kbn-xsrf': 'cypress',
+    },
+    auth: {
+      user: 'elastic',
```

Comment:
QQ: Should we use the `${env.ELASTICSEARCH_USERNAME}` and `${env.ELASTICSEARCH_PASSWORD}` here?